docs/news.rst

```diff
@@ -3,6 +3,269 @@
 Release notes
 =============
 
+.. _release-1.7.0:
+
+Scrapy 1.7.0 (2019-07-NN)
+-------------------------
+
+New features
+~~~~~~~~~~~~
+
+*   A ``process_request`` callback passed to the :class:`~scrapy.spiders.Rule`
+    constructor now receives the :class:`~scrapy.http.Response` object that
+    originated the request as its second argument (:issue:`3682`)
+
+    ``process_request`` callbacks that do not accept this second argument
+    are **deprecated** and will stop working in a future version of Scrapy.
+
+*   A new ``restrict_text`` parameter for the
+    :attr:`LinkExtractor <scrapy.linkextractors.lxmlhtml.LxmlLinkExtractor>`
+    constructor allows filtering links by linking text (:issue:`3622`,
+    :issue:`3635`)
+
+*   A new :setting:`FEED_STORAGE_S3_ACL` setting allows defining a custom ACL
+    for feeds exported to Amazon S3 (:issue:`3607`)
+
+*   A new :setting:`METAREFRESH_IGNORE_TAGS` setting allows overriding which
+    HTML tags are ignored when searching a response for HTML meta tags that
+    trigger a redirect (:issue:`1422`, :issue:`3768`)
+
+*   A new :reqmeta:`redirect_reasons` request meta key exposes the reason
+    (status code, meta refresh) behind every followed redirect (:issue:`3581`,
+    :issue:`3687`)
+
+*   The ``SCRAPY_CHECK`` variable is now set to the ``true`` string during runs
+    of the :command:`check` command, which allows :ref:`detecting contract
+    check runs from code <detecting-contract-check-runs>` (:issue:`3704`,
+    :issue:`3739`)
+
+*   A new scheduler priority queue,
+    :class:`scrapy.pqueues.DownloaderAwarePriorityQueue`, may be
+    :ref:`enabled <broad-crawl-scheduler-priority-queue>` for a significant
+    scheduling improvement on crawls targetting multiple web domains, at the
+    cost of no :setting:`CONCURRENT_REQUESTS_PER_IP` support (:issue:`3520`)
+
+    As a side effect, ``queuelib.PriorityQueue`` is **deprecated** as a valid
+    value for :setting:`SCHEDULER_PRIORITY_QUEUE` and will stop working in a
+    future version of Scrapy. Use :class:`scrapy.pqueues.ScrapyPriorityQueue`
+    instead to continue using the default scheduler priority queue
+    implementation.
+
+*   A new :attr:`Request.cb_kwargs <scrapy.http.Request.cb_kwargs>` attribute
+    provides a cleaner way to pass keyword arguments to callback methods
+    (:issue:`1138`, :issue:`3563`)
+
+*   A new :class:`~scrapy.http.JSONRequest` class offers a more convenient way
+    to build JSON requests (:issue:`3504`, :issue:`3505`)
```

Comment:
What do you think about moving most prominent features (visible to users) to the top? Maybe new queue, cb_kwargs and JSONRequest?